spec/controllers/organizations_controller_spec.rb

```diff
@@ -2,12 +2,55 @@
   let!(:organization) { Fabricate(:organization) }
   let(:member) { Fabricate(:member, organization: organization) }
   let(:user) { member.user }
+  let!(:second_organization) { Fabricate(:organization) }
 
   describe 'GET #index' do
     it 'populates and array of organizations' do
       get :index
 
-      expect(assigns(:organizations)).to eq([organization])
+      expect(assigns(:organizations)).to eq([organization, second_organization])
+    end
+  end
+
+  describe 'GET #index (search)' do
+    before do
+      second_organization.name = "Banco del tiempo Doe"
+      second_organization.city = "Sevilla"
+      second_organization.address = "Calle gloria"
+      second_organization.neighborhood = "La paz"
+      second_organization.save!
+      organization.neighborhood = "La paz"
+      organization.save!
+    end
```

Comment:
🙏🏼  let's leave an empty line between each code block (before, it, describe, ...)